compiler/rustc_expand/src/mbe/transcribe.rs

```diff
@@ -421,22 +421,22 @@ fn lockstep_iter_size(
 /// * `[ $( ${count(foo, 0)} ),* ]` will be the same as `[ $( ${count(foo)} ),* ]`
 /// * `[ $( ${count(foo, 1)} ),* ]` will return an error because `${count(foo, 1)}` is
 ///   declared inside a single repetition and the index `1` implies two nested repetitions.
-fn count_repetitions<'a>(
```

Comment:
why did you rename the lifetime? It's not clearly better to me

Reply:
Hehehehe, it was a "good intention". `'a` doesn't have a semantic value but neither does `'any`. 

Reverted